test-metadata.py

```diff
@@ -0,0 +1,40 @@
+"""Test script for resdk.resources.Metadata."""
```

Comment:
This is great to quickly test the new functionality but don't forget to remove it before merging.

Reply:
Removed.